assets/error.html

```diff
@@ -98,15 +97,22 @@
 
 		async function getExceptionData() {
 			const wnd = await fin.Window.getCurrent();
-			const { customData } = await wnd.getOptions();
+			const { customData: { title, error } } = await wnd.getOptions();
+			const errorTitleEl = document.querySelector('#error-title');
 			const errorMessageEl = document.querySelector('#error-message');
-
-			errorMessageEl.innerHTML = customData.error.stack;
+			
+			if (title) {
+				errorTitleEl.innerHTML = title;
+			} else {
+				errorTitleEl.remove();
+			}
+			
+			errorMessageEl.innerHTML = error;
 			
 			// Keep increasing the height of the error dialog until
 			// the error message is fully visible. This will ensure
 			// error dialogs are not too big in height.
-			let resizeSafetyCount = 30; // for safety, max 30 resizes (300px)
+			let resizeSafetyCount = 50; // for safety, max 50 resizes (500px)
```

Comment:
Is there any way we can just do this in one step (calculating how big it needs to be)? Also, is this really better than just scrolling? What happens if another error happens in the mean time? Have we tested with another app crashing before the first dialog goes away?

Reply:
1. AFAIK, there is no other way to know in advance how big HTML element will be. And IMO it is better than scrolling or having a dialog that's too big
2. Re: Another crash before goes way... this dialog is building on the previous behavior. It can be investigated and improved (if need to) in another PR to make sure we cover this case.

Reply:
👍 There is just something about making a bunch of tight API when part of the system is known to be messed up makes me queasy...

Reply:
Yeah, I agree, it doesn't look pretty, but the only alternative to this is under/over-sized error boxes which is also not pretty. This is, however, a "hidden not pretty" :)